scripts/pgdata.sh

```diff
@@ -0,0 +1,50 @@
+#!/hab/pkgs/core/bash/5.1/20210826055113/bin/bash
+#shellcheck disable=SC1128
+#
+shopt -s extglob
+
+exec 2>&1
+
+# *** WARNING ***
+# Please put potentially long-running and/or complex operations in the init hook rather
+# than the run hook until the issue described in
+#
+# https://github.com/habitat-sh/habitat/issues/1973
+#
+# has been resolved.
+# Currently, the Habitat `init` and `health_check` hooks run directly from the main loop
+# of the Habitat supervisor. If these hooks hang or take too long to run, they can block
+# execution of the supervisor.
+#
+
+
+#shellcheck disable=SC1091
+source /hab/svc/automate-postgresql/config/functions.sh
+
+mkdir -p /hab/svc/automate-postgresql/config/conf.d
+mkdir -p /hab/svc/automate-postgresql/var/pg_stat_tmp
+
+# Auto-detect pg data in the root of svc_data_path, where older versions of this plan had it
+if [[ -f "/hab/svc/automate-postgresql/data/PG_VERSION" ]]; then
+  echo "PGDATA detected in the root of the data path ( /hab/svc/automate-postgresql/data ), relocating it to /hab/svc/automate-postgresql/data/pgdata"
```

Comment:
is this if section needed, as in new installations it will always be automate-postgresql/data/pgdata

Reply:
removed it